_locales/pl/messages.json

```
@@ -0,0 +1,383 @@
+	},
+	"options_link_inventory_gift_subid": {
+		"message": "Prawidłowo określ ID paczki prezentu w twoim ekwipunku"
+	},
+	"options_enhancement_inventory_no_sell_reload": {
+		"message": "Nie odświeżaj ekwipunku podczas sprzedaży przedmiotu"
+	},
+	"options_enhancement_inventory_quick_sell": {
+		"message": "Dodaj przyciski szybkiej sprzedaży"
+	},
+	"options_enhancement_inventory_quick_sell_auto": {
+		"message": "Automatycznie akceptuj szybką sprzedaż"
+	},
+	"options_display_market_price_others": {
+		"message": "Wyświetl cenę rynkową przedmiotów w ekwipunkach innych użytkowników"
+	},
+	"options_enhancement_tradeoffer_url_items": {
+		"message": "Wspieraj parametry <code>for_item</code> i <code>my_item</code>"
+	},
+	"options_github_explanation": {
+		"message": "Zobacz wyjaśnienie na Githubie"
+	},
+	"options_enhancement_tradeoffer_no_gift_confirm": {
+		"message": "Zignoruj powiadomienia ofert wymiany \"ta transakcja wydaje się podejrzana\" i \"czy jesteś pewien, że jest to prezent?\""
+	},
+	"options_enhancement_tradeoffer_better_errors": {
+		"message": "Wyświetlaj lepsze komunikaty o błędach, zamiast tajemniczych numerów błędów Valve"
+	},
+	"options_enhancement_skip_agecheck": {
+		"message": "Pomiń stronę weryfikacji wieku"
+	},
+	"options_enhancement_skip_agecheck_description": {
+		"message": "Ustaw ciasteczka wieku i urodzin, aby zawsze były akceptowane"
+	},
+	"options_enhancement_no_linkfilter": {
+		"message": "Usuń wszystkie linki filtru linków"
+	},
+	"options_enhancement_market_ssa": {
+		"message": "Akceptuj SSA automatycznie na stronach rynku, ekwipunku i aktywacji kluczy"
+	},
+	"options_enhancement_hide_install_button": {
+		"message": "Schowaj przycisk \"Zainstaluj Steam\" w nagłówku"
+	},
+	"options_enhancement_hide_mobile_app_button": {
+		"message": "Schowaj baner \"Otwórz w Steam\""
+	},
+	"options_steamdb_rating": {
+		"message": "Wyświetl oceny SteamDB na stronach aplikacji"
+	},
+	"options_enhancement_appicon": {
+		"message": "Wyświetl ikony aplikacji na stronach sklepu i społeczności"
+	},
+	"options_hidden_achievements": {
+		"message": "Załaduj ukryte osiągnięcia i ich opisy"
+	},
+	"options_spoiler_achievements": {
+		"message": "Ukryj opisy za spoilerami"
+	},
+	"options_prevent_store_images": {
+		"message": "Zatrzymaj ładowanie obrazów w sklepie na stronach aplikacji"
+	},
+	"options_prevent_store_images_description": {
+		"message": "Niektórzy deweloperzy umieszczają megabajty obrazów w opisach gier, których załadowanie może zająć trochę czasu."
+	},
+	"options_enhancement_award_popup_url": {
+		"message": "Obsługa parametrów <code>award</code> w linkach do profili i plików udostępnionych"
+	},
+	"options_builtin_card_drops": {
+		"message": "Wyświetl całkowitą liczbę pozostałych do zdobycia kart na stronie odznak"
+	},
+	"options_builtin_queue_cheat": {
+		"message": "Dodaj przycisk, aby oszukać kolejkę odkrywania podczas wyprzedaży"

```

```suggestion
		"message": "Dodaj przycisk, aby wyczyścić kolejkę odkrywania podczas wyprzedaży"
```

Nie mamy odpowiednika "cheat" w tym kontekście, lepiej odnieść się do stanu.